docs/configuration/logging.md

```diff
@@ -107,8 +107,9 @@ The following example log4j2.xml is based upon the micro quickstart:
 
 > NOTE:
 > Although Druid shares the log4j configuration file task peon processes,
```

Comment:
Shares from whom to whom? Should it say "shares the log4j configuration file **between** task peon processes"?

Reply:
How is this for the write up?
>Although Druid shares the log4j configuration file amongst all services, including task peon processes, the appenders in this file WILL NOT take effect for loggers in the peon processes unless there is a console appender defined. Instead all the appenders for peons processes will be overwritten to be console appenders and use the log levels as defined in the configuration file. This means you can configure loggers at different logging level for task logs using log4j2.xml.

I think keeping the note is useful since it is something that isn't intuitive, but you know docs better than me so i'm happy to remove it it